templates/repo/home.tmpl

```diff
@@ -92,6 +92,20 @@
 							<a class="item" href="{{.RepoLink}}/_new/{{.BranchName | PathEscapeSegments}}/{{.TreePath | PathEscapeSegments}}">
 								{{.locale.Tr "repo.editor.new_file"}}
 							</a>
+							{{if and .EnableActions (not .UnitActionsGlobalDisabled) .CanWriteActions}}
+							<div class="item">
+								<i class="dropdown icon"></i>
+								<span class="text">{{.locale.Tr "repo.editor.new_workflow"}}</span>
+								<div class="menu">
+									<a class="item" href="{{.RepoLink}}/_new/{{.BranchName | PathEscapeSegments}}/.gitea/workflows/{{.TreePath | PathEscapeSegments}}">
+										Gitea
+									</a>
+									<a class="item" href="{{.RepoLink}}/_new/{{.BranchName | PathEscapeSegments}}/.github/workflows/{{.TreePath | PathEscapeSegments}}">
+										GitHub
+									</a>
+								</div>
+							</div>
```

Comment:
I don't think this is the best possible way of achieving this.
If we do it this way, I can see the feature requests like `Please also add this and that to the "new file" menu. And make it configurable!` pouring in.
What is probably a better idea is offering presets on the actual `new page` page that the user can select from what they want.